content/en/developers/metrics/metrics_type.md

```diff
@@ -0,0 +1,271 @@
+
+Each data point represents the number of requests—the "count" of requests—received during the 30 second interval. The metric then has the following shape:
+
+* `10` for the first 30 seconds
+* `20` for the second interval of 30 seconds
+* `0` for the last interval of 30 seconds
+
+Then this pattern starts again, since the `COUNT` is the absolute variation of the number of requests:
+
+{{< img src="developers/metrics/metric_types/count_metric.png" alt="Count Metric" responsive="true">}}
+
+Note: StatsD counters can show a decimal value within Datadog, since they are normalized over the flush interval to report units per second.
+
+Discover how to submit count metrics:
+
+* [With a custom Agent Check][1]
+* [With DogStatsD][2]
+* [With the Datadog API][3]
+
+**Note**: for a `COUNT` metric, when a `0` value is submitted, `null` is stored within Datadog.
+
+[1]: /developers/metrics/agent_metrics_submission/?tab=count
+[2]: /developers/metrics/dogstatsd_metrics_submission/#count
+[3]: /api/?lang=python#post-timeseries-points
+{{% /tab %}}
+{{% tab "RATE" %}}
+
+**The `RATE` metric type represents a normalized per-second variation of a metric's value over a defined time interval**, contrary to the `COUNT` metric type, which represents this variation with an absolute value.
+
+For instance, say that the `number.of.requests.rate` metric is reported every 30 seconds to Datadog with the `RATE` type for `server:web_1`.
+
+Each data point represents the "rate" of requests. The metric then has the following shape:
+
+* `0.33` for the first 30 seconds
+* `0.66` for the second interval of 30 seconds
+* `0` for the last interval of 30 seconds
+
+Then this pattern starts again, since the `RATE` is the normalized per-second variation of the number of requests:
+
+{{< img src="developers/metrics/metric_types/rate_metric.png" alt="Rate Metric" responsive="true">}}
+
+Discover how to submit rate metrics:
+
+* [With a custom Agent Check][1]
+* [With the Datadog API][2]
+
+**Note**: for a `RATE` metric, when a `0` value is submitted, `null` is stored within Datadog.
+
+[1]: /developers/metrics/agent_metrics_submission/?tab=rate
+[2]: /api/?lang=python#post-timeseries-points
+{{% /tab %}}
+{{% tab "GAUGE" %}}
+
+**The `GAUGE` metric type represents the value of a particular thing over time.** It is a snapshot of a value associated with a timestamp.
+
+For instance, say that the `number.of.requests.gauge` metric is reported every 30 seconds to Datadog with the `GAUGE` type for `server:web_1`.
+
+Each data point represents the overall number of requests received at a point in time. The metric then has the following shape:
+
+* `10` for the first 30 seconds
+* `30` for the second interval of 30 seconds
+* `30` for the last interval of 30 seconds
+
+Then this pattern starts again, but it increases over time since the `GAUGE` metric type keeps track of how many requests have already been received:
```

Comment:
this is an interesting example, it really shows what happens when you use `gauge` for this type of data. Here, I think it show that it's non-sensical to track your example with a `gauge` (the increasing value in Datadog can't really be used for anything).

So maybe we should have another example where we would not could a number of requests, but a "state", for example the number of users currently browsing on our website (or the number of active connexions to our web servers, or something similar), where it makes sense to use gauges, and it generally doesn't make sense to use counts/rates.

(^ all this is only useful to explain the difference between gauges and rates/counts, but not really to explain what happens with histograms, distribution metrics, etc)

Reply:
Talked about this off-line, this is solved.